src/test/kotlin/io/github/mojira/arisa/modules/HelpersTest.kt

```diff
@@ -0,0 +1,305 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import io.github.mojira.arisa.utils.RIGHT_NOW
+import io.kotest.assertions.arrow.either.shouldBeLeft
+import io.kotest.assertions.arrow.either.shouldBeRight
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeTypeOf
+import java.time.Instant
+
+private val throwable = Throwable(message = "generic error")
+private const val success = "success"
+private val NOW = Instant.now()
+private val A_SECOND_AGO = RIGHT_NOW.minusSeconds(1)
+
+class HelpersTest : StringSpec({
+    "toFailedModuleEither when given Throwable in Either should return FailedModuleResponse with that 1 throwable" {
+        val moduleEither = throwable.left().toFailedModuleEither()
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(FailedModuleResponse(listOf(throwable)))
+    }
+
+    /* commented out as I don't know yet whether this is a feature or not
+    "toFailedModuleEither when given success in Either should return the success in ModuleResponse" {
```

Comment:
This should probably exist :P

Reply:
k

Reply:
done in https://github.com/mojira/arisa-kt/pull/433/commits/7c5bddd90ca644a8f09e26c293a336d735ceb72f